src/o365/graph/commands/teams/teams-user-add.ts

```diff
@@ -0,0 +1,170 @@
+import auth from '../../GraphAuth';
+import config from '../../../../config';
+import commands from '../../commands';
+import GlobalOptions from '../../../../GlobalOptions';
+import {
+  CommandOption, CommandValidate
+} from '../../../../Command';
+import { GraphItemsListCommand } from '../GraphItemsListCommand';
+import Utils from '../../../../Utils';
+import { GroupUser } from '../o365group/GroupUser';
+import * as request from 'request-promise-native';
+
+const vorpal: Vorpal = require('../../../../vorpal-init');
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+  role?: string;
+  teamId: string;
+  userName: string;
+}
+
+class GraphTeamsUserAddCommand extends GraphItemsListCommand<GroupUser> {
```

Comment:
Since you're not using the `getAllItems` method, there is no need to use `GraphItemsListCommand`. Instead, we can use the `GraphCommand` base class.